tools/src/main/resources/ui/html/sql-recommendation.html

```diff
@@ -0,0 +1,172 @@
+          Reports
+        </a>
+        <div class="dash-nav-dropdown-menu">
+          <a href="raw.html" class="dash-nav-dropdown-item">Raw Data</a>
+          <a href="sql-recommendation.html" class="dash-nav-dropdown-item">Per-SQL Data</a>
+        </div>
+      </div>
+      <div class="dash-nav-dropdown show">
+        <a href="#!" class="dash-nav-item dash-nav-dropdown-toggle">
+          <i><svg width="1.05em" height="1.05em" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path d="M8 5V4M8 12V6M14.5 8C14.5 11.5899 11.5899 14.5 8 14.5C4.41015 14.5 1.5 11.5899 1.5 8C1.5 4.41015 4.41015 1.5 8 1.5C11.5899 1.5 14.5 4.41015 14.5 8Z" stroke="white"/>
+          </svg></i>
+          About </a>
+        <div class="dash-nav-dropdown-menu">
+          <a href="https://github.com/NVIDIA/spark-rapids" class="dash-nav-dropdown-item">GitHub</a>
+          <a href="https://nvidia.github.io/spark-rapids/" class="dash-nav-dropdown-item">
+            <i><svg width="1.05em" height="1.05em" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path d="M13.5 13.5V14H14V13.5H13.5ZM13.5 2.5H14V2H13.5V2.5ZM4 5H11.5V4H4V5ZM4 7H11.5V6H4V7ZM4 9H8V8H4V9ZM13.5 11H3.5V12H13.5V11ZM3.5 14H13.5V13H3.5V14ZM2 12.5C2 13.3284 2.67157 14 3.5 14V13C3.22386 13 3 12.7761 3 12.5H2ZM3.5 11C2.67157 11 2 11.6716 2 12.5H3C3 12.2239 3.22386 12 3.5 12V11ZM13.5 2H3.5V3H13.5V2ZM2 3.5V12.5H3V3.5H2ZM14 13.5V11.5H13V13.5H14ZM14 11.5V2.5H13V11.5H14ZM3.5 2C2.67157 2 2 2.67157 2 3.5H3C3 3.22386 3.22386 3 3.5 3V2Z" fill="white"/>
+            </svg></i> Docs </a>
+        </div>
+      </div>
+    </nav>
+  </div>
+  <div class="dash-app">
+    <header class="dash-toolbar">
+      <a href="#!" class="menu-toggle">
+        <i><svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path d="M6 3.5H14M6 11.5H14M6 7.5H14M2.5 2.5H4.5V4.5H2.5V2.5ZM2.5 10.5H4.5V12.5H2.5V10.5ZM2.5 6.5H4.5V8.5H2.5V6.5Z" stroke="black"/>
+        </svg></i></a>
+      <div class="tools">
+        <a href="https://github.com/NVIDIA/spark-rapids" class="tools-item">GitHub</a>
+      </div>
+    </header>
+    <main class="dash-content">
+      <div class="container-fluid">
+        <h1 class="dash-title">Per-SQL GPU Recommendation</h1>
+        <div class="row">
+          <div class="col-lg-8">
+            <div class="alert alert-warning alert-dismissible fade show" role="alert">
+              <svg width="1.2em" height="1.2em" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path d="M8 6.5V10M8 11.0001V12.0001M8 2L14.5 13.4983H1.5L8 2Z" stroke="black"/>
+              </svg>
+              <strong>Disclaimer!</strong><ul>
+              <li>Estimates provided by the Qualification tool are based on the currently supported "<em>SparkPlan</em>" or "<em>Executor Nodes</em>" used in the application. It currently does not look at the expressions or datatypes used.</li>
```

Comment:
we may want to update the text about not looking at expressions so say doesn't handle all expressions, probably needed elsewhere, perhaps handle under the other doc pr?

Reply:
I was planning to handle that in the upcoming PR #6007 so that I can reference the new section listing "parsed execs"